azure-fs-aks/Program.fs

```diff
@@ -0,0 +1,131 @@
+﻿module Program
+
+open Pulumi.FSharp
+open Pulumi.Azure.Core
+open Pulumi.Azure.ContainerService
+open Pulumi.Azure.ContainerService.Inputs
+open Pulumi.Azure.AD
```

Comment:
Nit: sort

Reply:
Haha. So you're one of those guys ;). I'll sort them.

Reply:
😈